ci/release/update-version.sh

```diff
@@ -53,11 +55,11 @@ for FILE in dependencies.yaml conda/environments/*.yaml; do
   done
 done
 
-
 # README.md update
 sed_runner "s/version == ${CURRENT_SHORT_TAG}/version == ${NEXT_SHORT_TAG}/g" README.md
```

Comment:
```suggestion
sed_runner "/version == / s/== .*\`/== ${NEXT_SHORT_TAG}\`/g" README.md
```

Suggesting the removal of the variable in the search pattern.